python/taichi/lang/__init__.py

```diff
@@ -603,6 +603,7 @@ def loop_unique(val, covers=None):
 vectorize = _ti_core.vectorize
 bit_vectorize = _ti_core.bit_vectorize
 block_dim = _ti_core.block_dim
+thread_idx = _ti_core.insert_thread_idx_expr
```

Comment:
Thanks a lot! 
Nits after a brief discussion with @k-ye  : 
We prefer to name `ti.thread_idx()` as `ti.global_thread_idx()` to make it self-explaining and avoid divergence from thread_idx() in the codebase.
```suggestion
global_thread_idx = _ti_core.insert_thread_idx_expr
```